swagger/v4/swagger.json

```diff
@@ -3541,7 +3541,7 @@
             "id": "7c660864-b50a-4b3c-b019-c37a3fba3b8e",
             "type": "display",
             "view_type": "paragraph",
-            "text": "Enter the supplies left in stock at the end of every month"
+            "text": "If you have more than 3 batches then add any additional batch stocks to batch 3 using the same date"
           },
```

Comment:
@olttwa Can you check if this is the intended value? I haven't tweaked anything related to questionnaires, and I suspect in #5212 the PR hadn't included the latest generated swagger JSON. This showed up when I regenerated it for the Bulk Import Docs.

Reply:
@tfidfwastaken acc to master branch, the expected text here is `"Enter the supplies left in stock at the end of every month"`. Can rollback your merged changes.

Reply:
Yeah, I saw that in the master branch too, but is the master branch correct? This `swagger.json` is generated from `specimen_layout.rb`, and currently the value of that is `If you have more than 3 batches...` as seen here: https://github.com/simpledotorg/simple-server/blob/f5bb61a2266eabbaaec9cfe4a8dcc33ba84f7e87/app/schema/api/v4/models/questionnaires/specimen_layout.rb#L220

So the source and generated file went out of sync at some point—I'd have assumed that the source file should be the one to go with, not the generated file. Can you confirm again?

Reply:
Okay, I went digging a bit more and this Slack thread suggests that it should be `"Enter the ..."`: https://simpledotorg.slack.com/archives/C01BYR9M933/p1690968264030099?thread_ts=1690904295.849239&cid=C01BYR9M933

The change #5223 made in response to that changed only the `en.yml` locale file and missed changing the `specimen_layout.rb`, which is why any subsequent PR that calls `rswag:generate` will create this incorrect diff (Example: [this one](https://github.com/simpledotorg/simple-server/pull/5242/files#diff-87e0f7763f4c24150621d28d7df5966c0e574b07507b039c7d8cdc9ab8e872afL3544-R3549)).

I'll make a separate PR that fixes this minor bug, and rebase this PR on that.